### ISSUE TITLE
Present Styling object in GEF editor

### ISSUE THREAD

**Issue by IanMayo** [debrief/limpet#314]
I'd like to be able to edit a dataset color, marker, marker size, etc.

Instead of pulling these properties into the Dataset object, I think an alternate strategy is to add a "Style" element to the Dataset figure.  I can get a custom icon drawn.  But, before then, could you add a selectable entity to the DatasetFigure object. When selected, then the Styling object for the Dataset will be shown in the properties window

**Comment by dinkoivanov** [comment]
I think that having a separate selectable element to handle styling might visually overload the editor. If possible I would explore the possibility to modify/extend the ExtendedPropertySheetPage or the AdapterFactoryContentProvider, so that for Dataset objects, it also lists the Styling object properties.
What do you think?

**Comment by IanMayo** [comment]
Combining them was my first thought - which would probably be easier if I generated the EMF.Edit & EMF.Editor sources.

But, then I thought about the Dataset object having a choice of Styling object. I imagined swapping Styling objects by right-clicking on the Styling child element & providing a "Replace with"... and then providing the list of alternate styling elements.

This is just a thought though.
